app/components/school-competencies-list.js

```diff
@@ -0,0 +1,26 @@
+import Component from '@glimmer/component';
+import { cached } from '@glimmer/tracking';
+import { TrackedAsyncData } from 'ember-async-data';
+import { map } from 'rsvp';
+import { sortBy } from 'ilios-common/utils/array-helpers';
+
+export default class SchoolCompetenciesListComponent extends Component {
+  @cached
+  get data() {
+    return new TrackedAsyncData(this.loadData(this.args.domains));
```

Comment:
This is a little bit under-tracked because the `children` of a `domain` changing won't trigger this to re-compute, however where this is used we re-render the page when saving from manage competencies so it doesn't matter. I just wanted to mention it as a possible gotcha in the future.